bindata/bootkube/manifests/configmap-csr-controller-ca.yaml

```diff
@@ -4,6 +4,8 @@ kind: ConfigMap
 metadata:
   name: csr-controller-ca
   namespace: openshift-config-managed
+  annotations:
+    "openshift.io/owning-component": "kube-apiserver"
```

Comment:
I claim this is owned by the kube-controller-manager which writes content to this location here: https://github.com/openshift/cluster-kube-controller-manager-operator/blob/master/pkg/operator/targetconfigcontroller/targetconfigcontroller.go#L724

While you may wish to try moving this part of rendering, do so as a followup because it's reasonably likely something will break.